src/main/java/com/github/wovnio/wovnjava/Headers.java

```diff
@@ -127,4 +128,29 @@ public boolean getIsValidRequest() {
         hreflangs.put(hreflangCodeDefaultLang, urlDefaultLang);
         return hreflangs;
     }
+
+    private boolean isIgnoredPath(String contextPath) {
+        ArrayList<String> ignorePaths = this.settings.ignorePaths;
+
+        if (ignorePaths.isEmpty()) {
+            return false;
+        }
+
+        for (String ignorePath : ignorePaths) {
+            if (ignorePath.isEmpty()) {
```

Comment:
I would not bother with this check. Or in Settings.java you could make a `normalizeIgnorePaths` method where you can remove the invalid paths, and maybe insert missing slashes